FirebaseCore.podspec

```diff
@@ -28,13 +28,14 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   s.source_files = 'FirebaseCore/Sources/**/*.[mh]'
   s.public_header_files = 'FirebaseCore/Sources/Public/*.h', 'FirebaseCore/Sources/Private/*.h'
   s.private_header_files = 'FirebaseCore/Sources/Private/*.h'
+  s.preserve_paths = 'Interop/CoreDiagnostics/Public/*.h'
```

Comment:
@paulb777  I has tried this approach for another SDK and noticed that `preserve_paths` are not added to the project by `pod gen`. Do you think there is a nice workaround for this? Will it be appropriate to add this file to sources as well?

Reply:
Switched from `preserve_paths` to `source_files`.